demos/basics/loading-numpy.ipynb

```diff
@@ -0,0 +1,923 @@
+{
```

Comment:
<blockquote>Some algorithms work with than just a feature vector associated with each node.</blockquote><p>"Some algorithms work with more than just a feature vector associated with each node."</p><p> </p><p>Maybe also worth mentioning here that the dimensions in the multidimensional features are <code>N x F x T</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1599/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
I'm not sure what F and T are there? In particular, if F = features/variates and T = timesteps, it should be N x T x F.

Reply:
Yep F = features/variates and T = timesteps. It says:

> For our square graph, we might have two time series of length three associated with each node

and I'm seeing
```
feature_tensors = np.array(
    [
        [[1.0, 1.0, 0.9], [-0.2, 0.1, 0.1]],
        [[2.0, 1.9, 2.1], [0.3, 0.31, 0.32]],
        [[3.0, 10.0, 3.0], [0.0, 0.0, 0.0]],
        [[4.0, 0.0, 1.0], [-0.5, -1.0, -3.0]],
    ],
    dtype=np.float32,
)
```
Isn't that 4 x 2 x 3? Which means it's N x F x T?

Reply:
Ah, yeah, I guess that's a typo.